substrate/primitives/trie/Cargo.toml

```diff
@@ -32,7 +32,7 @@ rand = { version = "0.8", optional = true }
 scale-info = { version = "2.11.1", default-features = false, features = ["derive"] }
 thiserror = { optional = true, workspace = true }
 tracing = { version = "0.1.29", optional = true }
-trie-db = { version = "0.28.0", default-features = false }
+trie-db = { version = "0.29.0", default-features = false }
```

Comment:
In this file trie-bench need to be upgraded to 0.39 (forgot to tell you :)

Reply:
Thank you! Solved here: [build: ⬆️ Upgrade trie-bench from 0.38.0 to 0.39.0](https://github.com/paritytech/polkadot-sdk/pull/3982/commits/6d0016afeadb642435754fd3ebc608e36ae454d3)